### PR TITLE
UCS/JENKINS: Enable the loopback interface as a tcp and testing resource

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Review by dmitrygx — APPROVED** [review]

**Comment by yosefe** [comment]
error seems related - 127.0.0.1:
```
[2021-01-20T20:09:37.755Z] [     INFO ] server listening on 127.0.0.1:46761
[2021-01-20T20:09:37.755Z] [1611173377.627396] [hpc-test-node-upstream-02:5609 :0]       rdmacm_cm.c:143  UCX  ERROR [cep 0x2e7d400 127.0.0.1:38742->127.0.0.1:46761 client Success]: rdma_resolve_route failed: No such device
[2021-01-20T20:09:37.755Z] /scrap/jenkins/workspace/ucx-7/contrib/../test/gtest/ucp/test_ucp_sockaddr.cc:225: Failure
```

File: test/gtest/ucp/test_ucp_sockaddr.cc [hunk 1]
```diff
@@ -135,6 +135,40 @@ class test_ucp_sockaddr : public ucp_test {
...
+        status = ucs_sock_ipstr_to_sockaddr(ipv4_addr,
+                                            (struct sockaddr_storage *)&lo_ipv4);
+        ASSERT_UCS_OK(status);
+
+        status = ucs_sock_ipstr_to_sockaddr(ipv6_addr,
+                                            (struct sockaddr_storage *)&lo_ipv6);
+        ASSERT_UCS_OK(status);
+
+        if (!ucs_sockaddr_cmp(m_test_addr.get_sock_addr_ptr(),
+                              (struct sockaddr *)&lo_ipv4, &status) ||
```
Comment (dmitrygx) [inline test/gtest/ucp/test_ucp_sockaddr.cc:160]: will it be static checker issue to expect that `status == UCS_OK`, but setting it for ipv4 comparison and then for ipv6 comparison? i.e. if ipv4 fails with `UCS_ERR_INVALID_PARAM` and then ipv6 comparison is `UCS_OK`, the assertion on line 163 will be successful, but it is unexpected

Comment (alinask) [reply test/gtest/ucp/test_ucp_sockaddr.cc:160]: @dmitrygx i'll split to 2 statuses then.

Comment (dmitrygx) [reply test/gtest/ucp/test_ucp_sockaddr.cc:160]: > @dmitrygx i'll split to 2 statuses then.

@alinask I think it is better
thanks

Comment (alinask) [reply test/gtest/ucp/test_ucp_sockaddr.cc:160]: thanks! 👍

**Review by dmitrygx — APPROVED** [review]

**Comment by alinask** [comment]
@yosefe can you please review?

**Review by yosefe — APPROVED** [review]
pls squash

**Comment by alinask** [comment]
the failure is https://github.com/openucx/ucx/issues/6195

**Comment by alinask** [comment]
bot:pipe:retest

**Pull request merged**